bundle/Command/GenerateProjectCommand.php

```diff
@@ -190,4 +193,50 @@ protected function cleanup(): void
             // Do nothing
         }
     }
+
+    private function activateGitHooks(): void
+    {
+        if (!class_exists(Config::class)) {
+            return;
+        }
+
+        /** @var \Symfony\Component\Filesystem\Filesystem $fileSystem */
+        $fileSystem = $this->getContainer()->get('filesystem');
+        $projectDir = $this->getContainer()->getParameter('kernel.project_dir');
+        $captainHookFiles = [
+            $projectDir . '/captainhook_disabled.json',
+            $projectDir . '/captainhook_enabled.json',
+        ];
+
+        if (!$fileSystem->exists($captainHookFiles)) {
+            return;
+        }
+
+        if ($this->questionHelper->ask(
+            $this->input,
+            $this->output,
+            $this->getConfirmationQuestion(
+                    'Do you want to use git hooks suitable for project development?',
+                    false
+                )
+        )
+        ) {
+            $fileSystem->symlink(
+                $fileSystem->makePathRelative(
+                    $projectDir,
+                    $projectDir
+                ) . '/captainhook_enabled.json',
+                $projectDir . '/captainhook.json');
+
+            return;
+        }
+
+        $fileSystem->symlink(
```

Comment:
Do we need this, considering by default this one is symlinked anyway?

Reply:
Removed in d2cf3fe